keras_cv/models/backbones/vit/vit_aliases.py

```diff
@@ -0,0 +1,401 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from keras_cv.models.backbones.vit.vit_backbone import ViTBackbone
+from keras_cv.utils.python_utils import classproperty
+
+
+class ViTTiny16Backbone(ViTBackbone):
+    def __new__(
+        cls,
+        include_rescaling=True,
+        input_shape=(224, 224, 3),
+        input_tensor=None,
+        **kwargs
+    ):
+        kwargs.update(
+            {
+                "include_rescaling": include_rescaling,
+                "input_shape": input_shape,
+                "input_tensor": input_tensor,
+            }
+        )
+        return ViTBackbone.from_preset("vittiny16", **kwargs)
+
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return {}
```

Comment:
Let's include the pre-trained presets here and in `presets_with_weights` for each alias

Reply:
Do you mind sharing what that would look like, exactly?

Reply:
Sure -- [this](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_aliases.py#L101) is a good example in MobileNet 😄

Reply:
Would it look like something as this?

```python
@classproperty
def presets(cls):
    """Dictionary of preset names and configurations."""
    return {
        "vittiny16_imagenet": copy.deepcopy(
            backbone_presets["vittiny16_imagenet"]
        ),
    }

@classproperty
def presets_with_weights(cls):
    """Dictionary of preset names and configurations that include
    weights."""
    return {
         "vittiny16_imagenet": copy.deepcopy(
              backbone_presets["vittiny16_imagenet"]
        ),
    }
```

Reply:
yep!